ingame_api/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```

Comment:
Since we don't have real achievements or stats, consider adding tests since this will be our only way of checking that this works properly